manifests/init.pp

```diff
@@ -1,9 +1,18 @@
+# @summary module to configure a systemd timer + script to backup postgres, mysql and ldap dataabases
+#
+# @param pigz_cpu_cores number of cpu cores used for backup compression
```

Comment:
I'm a fan of a newline. So
```suggestion
# @param pigz_cpu_cores
#   number of cpu cores used for backup compression
```

(and repeated elsewhere).

Reply:
ok, will do